### ISSUE TITLE
Question: Density for flyouts and drop downs

### ISSUE THREAD

**Issue by mdtauk** [microsoft/microsoft-ui-xaml#925]
Will the Date and Time Picker flyouts be resized when the control itself is in a CompactDensity?

How about ComboBoxes, MenuFlyouts, DropDowns, CommandBarFlyouts?

![image](https://user-images.githubusercontent.com/7389110/59958668-35383680-94a2-11e9-8f0d-58a28b133078.png)

![image](https://user-images.githubusercontent.com/7389110/59958675-3ff2cb80-94a2-11e9-8019-0de7adebd1a2.png)

![image](https://user-images.githubusercontent.com/7389110/59958683-4d0fba80-94a2-11e9-9126-fd178e986390.png)

**Comment by jevansaks** [comment]
> Will the Date and Time Picker flyouts be resized when the control itself is in a CompactDensity?

@mdtauk yes they're supposed to be, but IIRC there was a platform issue where they were hard coded. I *think* we made some progress on this in Windows 1903 but it's not perfect. We'll take another look at this post WinUI 3.0.

**Comment by mdtauk** [comment]
Following a discussion with @chigy, I have submitted it as a bug.  #1148

**Comment by mdtauk** [comment]
Flyouts are still not given any support for Compact Sizing when spawned by a parent control using compact sizing

**Comment by niels9001** [comment]
Density is not currently well supported across the platform with WinUI 3 and parts of the Density styles have been removed in the migration from WinUI 2 to WinUI 3.
